tests/plan.spec.ts

```diff
@@ -369,6 +370,16 @@ test.describe("preview", async () => {
       await expect(page.getByTestId("plan-preview-title")).toHaveText("Preview plan");
     });
   });
+  test("fixed tariff: show prices", async ({ page }) => {
+    await restart(CONFIG_FIXED_TARIFF);
+    await page.goto("/");
+
+    const lp1 = await page.getByTestId("loadpoint").first();
+    await lp1.getByTestId("charging-plan").getByRole("button", { name: "none" }).click();
+
+    const modal = await page.getByTestId("charging-plan-modal");
+    await expect(modal.getByTestId("tariff-value")).toHaveText("Energy price40.0 ct/kWh");
```

Comment:
Ich schreibe das gerne so. das macht inhaltlich zwar genau das Gleiche und hat mehr Zeichen, aber das machts einfacher lesbar. Gerade wenn es mehr Text wird.

```suggestion
    await expect(modal.getByTestId("tariff-value")).toHaveText(["Energy price", "40.0 ct/kWh"].join(""));
```